rotkehlchen/tests/unit/test_inquirer.py

```diff
@@ -277,3 +281,53 @@ def test_find_usd_price_via_second_oracle(inquirer):
     assert price == expected_price
     for oracle_instance in inquirer._oracle_instances[0:2]:
         assert oracle_instance.query_current_price.call_count == 1
+
+
+@pytest.mark.parametrize('use_clean_caching_directory', [True])
+@pytest.mark.parametrize('should_mock_current_price_queries', [False])
+def test_price_underlying_tokens(inquirer, globaldb):
+    aave_weight, link_weight, crv_weight = FVal('0.6'), FVal('0.2'), FVal('0.2')
+    address = make_ethereum_address()
+    token = CustomEthereumToken(
+        address=address,
+        decimals=18,
+        name='Test',
+        symbol='YAB',
+        underlying_tokens=[
+            UnderlyingToken(address=A_AAVE.ethereum_address, weight=aave_weight),
+            UnderlyingToken(address=A_LINK.ethereum_address, weight=link_weight),
+            UnderlyingToken(address=A_CRV.ethereum_address, weight=crv_weight),
+        ],
+    )
+    globaldb.add_asset(
+        asset_id=ethaddress_to_identifier(address),
+        asset_type=AssetType.ETHEREUM_TOKEN,
+        data=token,
+    )
+
+    aave_price = inquirer.find_usd_price(A_AAVE) * aave_weight
+    link_price = inquirer.find_usd_price(A_LINK) * link_weight
+    crv_price = inquirer.find_usd_price(A_CRV) * crv_weight
+
+    price = inquirer.find_price(EthereumToken(address), A_USD)
+    assert price == FVal(aave_price + link_price + crv_price)
+
+
+def test_find_lp_token_price(inquirer, globaldb, ethereum_manager):
```

Comment:
does hit test mock prices? Also add a simple docstring in the test to see what it does. Or I guess you should at least mention in the name that it's uniswap v2 lp